.github/workflows/stress.yml

```diff
@@ -28,6 +28,9 @@ name: Stress
 
 on: [push, pull_request]
 
+env:
+  OT_COMMIT: "0d8a99bd0573227f0751e18b584b069183d1eb3f"
```

Comment:
nit: give a documentation for this commits, or if possible,  create a tag for it.

Reply:
What about adding a date comment. I think it's infeasible to use a tag whenever the commit change.

Reply:
date added